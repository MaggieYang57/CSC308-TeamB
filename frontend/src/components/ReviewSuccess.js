import React from "react";
import { Link } from "react-router-dom";
import "bootstrap/dist/css/bootstrap.min.css";
import "../css/Signup.css";

require('dotenv').config()
const backendHostURL = process.env.REACT_APP_BACKEND_HOST_URL
let hikeIdPath;

if (backendHostURL === "http://localhost:3001")
  hikeIdPath = window.location.pathname.split('/')[2]
else if (backendHostURL === "https://slo-hikes-backend.herokuapp.com")
  hikeIdPath = window.location.href.split('/')[4]

function ReviewSuccess(props) {
  return (
    <div className="signup-form">
      <div id="title-signup">
        <b
          className="text text-center "
          style={{ fontSize: 40, color: "#2C6674" }}
        >
          Your review is added!
        </b>
      </div>
      <p style={{ marginTop: 20 }}>
        Back to <Link to={`/hike/${hikeIdPath}`}> hike page</Link>
      </p>
    </div>
  )
}

export default ReviewSuccess;
