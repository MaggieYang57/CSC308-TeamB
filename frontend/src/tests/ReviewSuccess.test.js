import React from 'react';
import { MemoryRouter } from 'react-router';
import renderer from 'react-test-renderer';
import ReviewSuccess from '../components/ReviewSuccess';

it('renders correctly', () => {
  const tree = renderer
    .create(<MemoryRouter> <ReviewSuccess /> </MemoryRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});