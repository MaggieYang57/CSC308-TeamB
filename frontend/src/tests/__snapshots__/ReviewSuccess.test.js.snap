// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  " ",
  <div
    className="signup-form"
  >
    <div
      id="title-signup"
    >
      <b
        className="text text-center "
        style={
          Object {
            "color": "#2C6674",
            "fontSize": 40,
          }
        }
      >
        Your review is added!
      </b>
    </div>
    <p
      style={
        Object {
          "marginTop": 20,
        }
      }
    >
      Back to 
      <a
        href="/"
        onClick={[Function]}
      >
        home page
      </a>
    </p>
  </div>,
  " ",
]
`;
